### DISCUSSION TITLE
Allow SNS HTTPS subscriptions

### DISCUSSION THREAD

**Discussion by TimPetricola** [runs-on/runs-on#54] [Ideas]
Having an email subscription for SNS notifications (RunsOn errors, RunsOn costs) is good. But we would like to instead subscribe via an HTTPS endpoint.

Our usecase is to forward these notifications to Datadog, through [their SNS integration](https://docs.datadoghq.com/integrations/amazon_sns/).

Adding a new subscription from the AWS console works perfectly fine, but I want to make sure that we won't lose this forwarding when we update RunsOn (or install a new major version).

Would it be possible to add an optional `SNSNotificationHTTPSEndpoint` parameter for such usecase?

**Comment by crohr** [comment]
This will land in https://github.com/runs-on/runs-on/pull/56

**Comment by crohr** [comment]
Implemented in v2.2.0
